tiflash/tiflash-configuration.md

```diff
@@ -155,30 +155,29 @@ delta_index_cache_size = 0
     ## The following configuration item only takes effect for the TiFlash disaggregated storage and compute architecture mode. For details, see documentation at https://docs.pingcap.com/tidb/dev/tiflash-disaggregated-and-s3.
     # disaggregated_mode = tiflash_write # The supported mode is `tiflash_write` or `tiflash_compute.
 
-## Multiple TiFlash nodes elect a master to add or delete placement rules to PD,
-## and the configurations in flash.flash_cluster control this process.
-[flash.flash_cluster]
-    refresh_interval = Master regularly refreshes the valid period.
-    update_rule_interval = Master regularly gets the status of TiFlash replicas and interacts with PD.
-    master_ttl = The valid period of the elected master.
-    cluster_manager_path = The absolute path of the pd buddy directory.
-    log = The pd buddy log path.
-
 [flash.proxy]
-    addr = The listening address of proxy. If it is left empty, 127.0.0.1:20170 is used by default.
-    advertise-addr = The external access address of addr. If it is left empty, "addr" is used by default.
-    data-dir = The data storage path of proxy.
-    config = The configuration file path of proxy.
-    log-file = The log path of proxy.
-    log-level = The log level of proxy. "info" is used by default.
-    status-addr = The listening address from which the proxy pulls metrics | status information. If it is left empty, 127.0.0.1:20292 is used by default.
-    advertise-status-addr = The external access address of status-addr. If it is left empty, "status-addr" is used by default.
+    ## The listening address of proxy. If it is left empty, 127.0.0.1:20170 is used by default.
+    addr = "127.0.0.1:20170"
+    ## The external access address of addr. If it is left empty, "addr" is used by default.
+    advertise-addr = ""
```

Comment:
If use 127.0.0.1 as advertise addr, peers on other nodes can't connect to the current node, should we emphasis that?

Reply:
@JaySon-Huang PTAL